src/components/Sidebar/Sidebar.js

```diff
@@ -1,14 +1,25 @@
-import React from "react";
-import { Button } from "../Button/Button";
-import { SidebarStyle } from "./SidebarStyle";
+import React from 'react';
+import { Link, useRouteMatch } from 'react-router-dom';
+import { Button } from '../Button/Button';
+import { SidebarStyle } from './SidebarStyle';
 
 export const Sidebar = () => {
+  const { url } = useRouteMatch();
+
   return (
     <SidebarStyle>
-      <Button />
-      <Button />
-      <Button />
-      <Button />
+      <Link to={`${url}/gender-identity`}>
+        <Button text="gender identity" />
+      </Link>
+      <Link to={`${url}/minority-group`}>
+        <Button text="minority groups" />
+      </Link>
+      <Link to={`${url}/previous-bootcamp`}>
+        <Button text="dev bootcamp" />
+      </Link>
+      <Link to={`${url}/employment-status`}>
+        <Button text="employment status" />
+      </Link>
```

Comment:
Question about the usage of the button, we currently don't really use the onClick function which is the main thing of the button, why don't we just style the link to looks like a button?

Reply:
I agree with this. Since we are linking to pages, we should be using the a tag (link in this case). There is no need for a button inside the a tag since buttons should be used to trigger inline functionality

Reply:
Good point! I will remove the button and put in text for now. In the CohortPage styling story, I will style the link to look like a button.